lib/iris/tests/unit/fileformats/netcdf/saver/test_Saver__ugrid.py

```diff
@@ -178,7 +178,7 @@ def mesh_location_size(mesh, location):
 
 
 # A simple "standard" test mesh for multiple uses, which we can use for cubes
-# that *share* a mesh (since we don't support mesh equality).
+# that ** a mesh (since we don't support mesh equality).
```

Comment:
Not sure what this is meant to mean?

"... we can use for cubes that ** a mesh..."

Reply:
Good spot -- definitely a mistake ! 🤭